socorro/external/crashstorage_base.py

```diff
@@ -652,6 +652,120 @@ def new_crashes(self):
 
 
 #==============================================================================
+class MigrationCrashStorage(FallbackCrashStorage):
+    required_config = Namespace()
+    required_config.add_option(
+        'date_threshold',
+        default="150401",
```

Comment:
there are many places in socorro config where fat fingering would cause problems.  There has not been a history of writing code that verifies values coming in from config as being of the correct type or domain.  I don't see a particularly compelling reason to start do such verifications in this location.